napari/_qt/layer_controls/qt_labels_controls.py

```diff
@@ -160,24 +162,37 @@ def __init__(self, layer):
             layer,
             'zoom',
             Mode.PAN_ZOOM,
-            tooltip=trans._('Pan/zoom mode (Space)'),
             checked=True,
         )
-        self.pick_button = QtModeRadioButton(
-            layer, 'picker', Mode.PICK, tooltip=trans._('Pick mode (L)')
+        action_manager.bind_button(
+            'napari:activate_picker_mode', self.panzoom_button
+        )
+
+        self.pick_button = QtModeRadioButton(layer, 'picker', Mode.PICK)
+        action_manager.bind_button(
+            'napari:activate_picker_mode', self.pick_button
         )
-        self.paint_button = QtModeRadioButton(
-            layer, 'paint', Mode.PAINT, tooltip=trans._('Paint mode (P)')
+
+        self.paint_button = QtModeRadioButton(layer, 'paint', Mode.PAINT)
+        action_manager.bind_button(
+            'napari:activate_paint_mode', self.paint_button
         )
+
         self.fill_button = QtModeRadioButton(
             layer,
             'fill',
             Mode.FILL,
-            tooltip=trans._(
-                "Fill mode (F) \nToggle with {shortcut}",
+        )
+        action_manager.bind_button(
+            'napari:activate_fill_mode',
+            self.fill_button,
+            extra_tooltip_text=trans._(
+                "Toggle with {shortcut}",
                 shortcut=Shortcut("Control"),
             ),
         )
+        # don't bind with action manager as this would remove "Toggle with {shortcut}"
```

Comment:
I don't understand the comment @Carreau 🤔, could you explain :)

Reply:
Remainders of earlier commit, removed.